bats/tests/common-setup.bash

```diff
@@ -23,6 +23,7 @@ _common_setup_file() {
   fi
   # clone or update the sources
   if [ -d $PYRSIA_TEMP_DIR/.git ]; then
+    log DEBUG "Merging Pyrsia changes into existing repo, repo dir: ${PYRSIA_TARGET_DIR}, repo: ${GIT_REPO}, branch: ${GIT_BRANCH}"
```

Comment:
It should probably log `PYRSIA_TEMP_DIR` instead of `PYRSIA_TARGET_DIR`?

```suggestion
    log DEBUG "Merging Pyrsia changes into existing repo, repo dir: ${PYRSIA_TEMP_DIR}, repo: ${GIT_REPO}, branch: ${GIT_BRANCH}"
```

Reply:
@tiainen  yes, you're right, updating.